package-lock.json

```diff
@@ -4363,6 +4368,19 @@
                 "url": "https://opencollective.com/typescript-eslint"
             }
         },
+        "node_modules/@vladfrangu/async_event_emitter": {
+            "version": "1.0.1",
+            "resolved": "https://registry.npmjs.org/@vladfrangu/async_event_emitter/-/async_event_emitter-1.0.1.tgz",
+            "integrity": "sha512-rkcSxwTBXQN6fyHygfcJBz5Zeq1DOZ1t8h+yxHu8iVo9CJkVGFboWPW6P+ZqsxGx0v92kUltNTSP/hUf1IlBiw==",
+            "dependencies": {
+                "@types/uuid": "^8.3.4",
+                "uuid": "^8.3.2"
```

Comment:
maybe things got better, but in general generating uuids is quite slow (especially through this package) and adding unnecessary perf costs on such low level as event emitter seems unwanted... maybe simple counter is enough? :]

Reply:
...well I overcomplicated myself with that. Give me a minute, let me fix that